### PR TITLE
Global Loader

### PR DESCRIPTION
**Pull request by AdithyaShaj**
1. Implemented global loader on initial render
2. used common customtable component for DC Open Orders
3. fixed refresh button style issues in home page

### CODE REVIEW

**Review by thasnim-cm — APPROVED** [review]

File: src/components/homePage.tsx [hunk 1]
```diff
@@ -172,89 +180,104 @@ const HomePage = () => {
...
+            </div>
+          </div>
+          {isLoading ? (
+            <Loader />
+          ) : (
+            <div className="flex flex-wrap gap-[10px]">
+              <HomeCard
+                title={
+                  <CardTitle
+                    title={"Avg Orders Per Min"}
```
Comment (sarath-sasi-panickaveetil) [inline src/components/homePage.tsx]: We can add these as appConstants.js. Same for all HomeCard components coming after this.

File: src/components/homePage.tsx [hunk 2]
```diff
@@ -172,89 +180,104 @@ const HomePage = () => {
...
-          <HomeCard
-            title={
-              <CardTitle
-                title={"Avg Orders Per Min"}
-                icon={avgOrdersPerMinIcon}
+        <div className="home-page p-6 box-border">
+          <div className="flex sm:flex-row justify-between mb-4">
+            <div className="flex items-center font-helvetica">
+              <span className="text-lg text-[#F2F2F2] font-bold mr-4">
+                Dashboard
```
Comment (sarath-sasi-panickaveetil) [inline src/components/homePage.tsx]: add this in appConstants

**Review by sarath-sasi-panickaveetil — APPROVED** [review]
Looks good to me

File: src/components/globalLoader.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import CustomImage from "./common/customimage";
+import GlobalLoaderIcon from "../assets/loader.gif";
+
+interface LoaderProps {
+  className?: string;
+}
+
+const GlobalLoader: React.FC<LoaderProps> = ({ className }) => {
```
Comment (raj-kumar-litmus) [inline src/components/globalLoader.tsx:9]: `const GlobalLoader = (props: LoaderProps) => {`

**Review by sarath-sasi-panickaveetil — APPROVED** [review]
Looks good to me

**Pull request merged**